poppy/poppy_core.py

```diff
@@ -2104,15 +2136,20 @@ def getPhasor(self,wave):
         else:
             return self.phasor
 
+    def getPhasor(self,wave):
+        warnings.warn("getPhasor is deprecated; use get_phasor instead", DeprecationWarning)
+        return self.get_phasor(wave)
+
+
     def display(self, nrows=1, row=1, what='intensity', crosshairs=True, ax=None, colorbar=True,
```

Comment:
why only set _vmax_ of OPD? _opd_vmin_ might be handy and transmission _vmax_/_vmin_, could be useful for viewing non-binary masks.